heuristic/local_search/loading_order.py

```diff
@@ -0,0 +1,12 @@
+from numpy.random import RandomState
+
+from heuristic.classes import Solution
+
+
+def loading_order(current: Solution, rnd_state: RandomState) -> Solution:
```

Comment:
Weird spacing (that's a general issue here - layout isn't caught by most checkers, so you need to come up with sensible spacing yourself).